src/HttpHeader.h

```diff
@@ -11,6 +11,7 @@
 
 #include "anyp/ProtocolVersion.h"
 #include "base/LookupTable.h"
+#include "http/ContentLengthInterpreter.h"
```

Comment:
This `#include` should not be needed. If ContentLengthInterpreter is missing from `http/forward.h` or equivalent, please add it. If there is no good `forward.h` file to add it to, forward-declare it here (or add the missing file).